webClient/src/app/core/menu-bar/menu-bar.config.ts

```diff
@@ -157,13 +158,15 @@ export const MENU = [
                 name: 'Open Directory',
```

Comment:
@NakulManchanda  Additionally, now that we have a few different shortcuts, yet no documentation yet... can you please add the corresponding hotkey to the title of the action? Similar to this,
![image](https://user-images.githubusercontent.com/20528015/66782931-d6f5f000-eea4-11e9-95d1-2225ed63691a.png)

This was customly requested by our UX team as well.

Reply:
Oh sorry, you said you did this.